projects/prysm-bn-validator-api.md

```diff
@@ -0,0 +1,66 @@
+# Prsym Beacon Node - Validator API
+
+Implementing the standard Ethereum Beacon API for validator clients in the Prysm beacon node
+
+## Motivation
+
+In production, the Prysm beacon node exposes a GRPC API for consumption by the Prysm validator.
+
+As a result of the standard Ethereum [spec](https://ethereum.github.io/beacon-APIs/#/Validator), this implementation is incompatible with other validator clients.
```

Comment:
might be worth clarifying that historically it was grpc and the standard is supposed to be REST..

Reply:
updated, quoted from a link issue in prysm